docs/applications/monitoring/index.rst

```diff
@@ -1,11 +1,24 @@
 .. px-app:: monitoring
 
-#####################################
-monitoring — Chronograf monitoring UI
-#####################################
+#########################################
+monitoring — Phalanx monitoring framework
+#########################################
+
+Monitoring implements several components.  The ``telegraf`` component is expected to run almost everywhere (and therefore the app will be enabled almost everywhere), and will send its data to a central InfluxDBv2 server.
```

Comment:
I don't like this design of merging the server and the client into a single application so that we have to enable that application everywhere but with most of the application disabled. Is there any way that we can go back to the way it was working before where the server and client are separate applications? That's much clearer about where the server is running.

Reply:
You're right.  I've done that now.

We can't merge the two because template variables are global.  Angelo is working up a PR, but not clear how lively upstream is.  If we get it accepted, great.  If not, having separate telegraf/-ds applications isn't the worst thing.